settings/DevHome.Settings/ViewModels/ExtensionsViewModel.cs

```diff
@@ -84,7 +86,9 @@ private void DisplaySettings()
                 continue;
             }
 
-            var setting = new Setting("Plugins/" + pluginWrapper.PackageFullName, pluginWrapper.PackageFullName, pluginWrapper.Name, string.Empty, string.Empty, true);
+            var hasSettingsProvider = pluginWrapper.HasProviderType(Microsoft.Windows.DevHome.SDK.ProviderType.Settings);
+
+            var setting = new Setting("Plugins/" + pluginWrapper.PackageFullName, pluginWrapper.PackageFullName, pluginWrapper.Name, string.Empty, string.Empty, true, hasSettingsProvider);

```

Comment:
FYI: I change this reference to pluginWrapper.PackageFullName in my change, will be a conflict for whoever goes in second.

Reply:
Fixed.